site/en/blog/renderingng-architecture/index.md

```diff
@@ -55,7 +55,7 @@ and [extensibility](/blog/renderingng/#extensibility-the-right-tools-for-the-job
 ### Rendering pipeline structure
 
 {% Img
-src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/jsWIjoou2iaN6ORvMfvM.jpeg",
+src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/zbb7OLj0rB1Hq57khU2g.jpeg",
```

Comment:
Please also reorder lines 85 and 86 from this file (not sure why I can't add a comment down there, but anyway..)